### DISCUSSION TITLE
How to exit?

### DISCUSSION THREAD

**Discussion by MasterCraft6969** [coder/code-server#6426] [Q&A] [answered]
After I ran it and finished my session. How do I end the vscode server session?

**Comment by code-asher** [comment, marked as answer]
Is this specifically VS Code's server or code-server?

For code-server it will depend on how you ran it, but keep in mind 
code-server is meant to be ran persistently.

**Reply by MasterCraft6969** [reply]
I got an answer from the people at the Termux discord server but forgot to close this issue. Apologies
